pkgs/cronet_http/android/build.gradle

```diff
@@ -2,14 +2,14 @@ group 'io.flutter.plugins.cronet_http'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.7.21'
```

Comment:
All of these types of upgrades should be done in a separate PR, no?

Reply:
I'm planning to reorg workflows and examples for the repo. These are necessary for the current setup. `1.6.10` is already outdated for the latest Flutter.

Reply:
OK, feel free to set these to any reasonable values.